pandas/_libs/tslibs/period.pyx

```diff
@@ -1217,6 +1217,32 @@ cdef class _Period(object):
 
     @property
     def day(self):
+        """
+        Return the number of days of particular month of a given Period.
+
+        This attribute returns the total number of days of given month on which
+        the particular date occurs.
+        
+        Returns
+        -------
+        Int
+        Number of days till particular date
+
+        See also
+        --------
+        Period.dayofweek
+            Return the day of the week
+
+        Period.dayofyear
+            Return the day of the year
+
+        Examples
+        --------
+        >>> import pandas as pd
```

Comment:
No need for `import pandas as pd` for any docstrings as it is implied (nor `import numpy as np`)

Reply:
sorry my mistake